### PR TITLE
Fix side number for file/folder created at a path used in the past

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

**Comment by nono** [comment]
[Mauvais numéro pour le side d'un fichier/dossier créé pour un chemin qui a été utilisé dans le passé](https://trello.com/c/w7P9wAkR/1355-mauvais-num%C3%A9ro-pour-le-side-dun-fichier-dossier-cr%C3%A9%C3%A9-pour-un-chemin-qui-a-%C3%A9t%C3%A9-utilis%C3%A9-dans-le-pass%C3%A9)

File: core/merge.js [hunk 1]
```diff
@@ -154,7 +154,9 @@ class Merge {
     }
     doc.sides[side] = ++rev
     doc._rev = info.rev
-    return this.pouch.put(doc)
+    const result = await this.pouch.put(doc)
+    delete doc._rev
```
Comment (taratatach) [inline core/merge.js]: What's the point of removing the doc's revision here? We're not returning `doc` or reusing it so I assume we don't really care what its value is.

Comment (nono) [reply core/merge.js]: Some unit tests failed when `_rev` is added to the doc.

Comment (taratatach) [reply core/merge.js]: Wouldn't that be a problem with how the tests were written rather than the implementation in this case?

**Comment by taratatach** [comment]
We'll wait for the stable release before merging this PR.

File: core/merge.js (hunk 1 above)
Comment (taratatach) [reply core/merge.js]: OK. Another argument for a good refactoring of Merge :).

**Pull request merged**